CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+CHANGES IN VERSION 1.17.0 (FROM 1.16.0)
+===================================
+
+Appium 1.17.0 is a minor release.
+
+#### General
```

Comment:
One thing to put into General:
- Active driver sessions are now properly cleanup upon main Appium process termination: https://github.com/appium/appium/pull/13913